src/redux/greeting/greetingSlice.js

```diff
@@ -0,0 +1,41 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  greeting: [],
+  loading: false,
+  error: null,
+};
+
+export const fetchGreeting = createAsyncThunk(
+  'greeting/fetchGreeting',
+  async () => {
+    try {
+      const response = await axios.get('http://127.0.0.1:3000');
```

Comment:
- [ ] Great job so far @Kwaku28 There is one change I will suggest you Implement and you'll be on your way to the next milestone  :wink: 
- [ ] Currently, there is an error in the Frontend App that is preventing me from viewing a randomly generated greeting and it looks like you haven't passed the complete URL link here. I noticed that the URL for a `random_greeting` in your Backend is this `http://127.0.0.1:3000/greetings/random_greeting` so kindly fix that as well and ensure that the two apps work as intended without any errors :wink: 
- Error Message
![image](https://github.com/Kwaku28/hello-react-front-end/assets/102699958/bac86ae8-34ed-4a00-8838-5fa46d16e8b3)